### ISSUE TITLE
Add a way to link carer tickets for admissions

### ISSUE THREAD

**Issue by marksteward** [emfcamp/Website#449]

**Comment by marksteward** [comment]
This requires:

- [ ] A table for the CarerTicket association
- [ ] An admin interface to set it (and potentially issue the carer ticket too)
- [ ] A tweak to the check-in interface to show that either of these tickets should be treated as associated, just as those from the same payment are.

**Comment by russss** [comment]
I don't think this is especially important at this stage, not sure what the use case is.
